ecommerce/migrations/0006_alter_productimage_options_and_more.py

```
@@ -0,0 +1,68 @@
+    operations = [
+        migrations.AlterModelOptions(
+            name='productimage',
+            options={'ordering': ['created_at'], 'verbose_name': 'Product Image', 'verbose_name_plural': 'Product Images'},
+        ),
+        migrations.RenameField(
+            model_name='product',
+            old_name='stock',
+            new_name='available_stock',
+        ),
+        migrations.RenameField(
+            model_name='product',
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)